geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/utils/GeoJsonInference.scala

```diff
@@ -0,0 +1,292 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.tools.utils
+
+import java.io.InputStream
+import java.{lang => jl}
+import java.time.format.DateTimeParseException
+
+import com.vividsolutions.jts.geom._
+import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
+import com.fasterxml.jackson.core.{JsonFactory, JsonParser, JsonToken}
+import com.fasterxml.jackson.core.JsonToken._
+import org.geotools.feature.AttributeTypeBuilder
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder
+import org.opengis.feature.`type`.AttributeDescriptor
+import org.opengis.feature.simple.SimpleFeatureType
+import org.locationtech.geomesa.utils.io.PathUtils
+import org.locationtech.geomesa.utils.io.fs.FileSystemDelegate
+import org.locationtech.geomesa.utils.geotools.{AttributeSpec, CRS_EPSG_4326}
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureSpecConfig.TypeNamePath
+import org.locationtech.geomesa.utils.text.DateParsing
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+import scala.reflect.ClassTag
+
+object GeoJsonInference {
+
+  val FieldsPath = "fields"
+  val OptionsPath = "options"
```

Comment:
these might already be defined as constants somewhere else you could use, although maybe not available on the current classpath...